test/views/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+var assert = require('assert');
+var helpers = require('../../../views/helpers');
+
+
+describe('views.helpers', () => {
+  describe('#contrastColor', () => {
+    it('should return "white" when the value is not present', () => {
+      assert.equal('white', helpers.contrastColor(null));      
+      assert.equal('white', helpers.contrastColor(''));
+    });
+
+    it('should return "black" when value is "#FFFFFFF"', () => {      
```

Comment:
test description does not matches what's tested -> description should be on a higher abstraction level than the code (otherwise it doesn't provide any value), better:

```
describe('#contrastColor', () => {
  describe('rgb color parsing', () => {
     it('supports '#' as prefix', => {});
     it('supports no prefix', => {});
  }
  it('generates contrast colors', () => {
     assert.equal('black', helpers.contrastColor("#FFFFFF"));
     assert.equal('white', helpers.contrastColor("#00000"));
  })
```